py/tests/unit/with_runtime_sparkling/test_word2vec_tokenizer.py

```diff
@@ -0,0 +1,45 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pysparkling.ml import H2OWord2VecTokenizer
+
+def testTokenizerTransform(craiglistDataset):
```

Comment:
Would it be possible to add tests testing compatibility of this tokenizer and H2O's Word2Vec?

Reply:
I was planning to add such test once we expose Word2Vec in SW, otherwise we would need to use JAVA api in tests which would break the client-less tests approach

Reply:
Test will be added as part of W2V integ test suite once it is exposed